utils_cv/detection/bbox.py

```diff
@@ -139,4 +144,46 @@ def __repr__(self):
             if self.label_name == str(self.label_idx)
             else self.label_name
         )
-        return f"{{{str(self)} | <{name}> | label:{self.label_idx} | path:{self.im_path}}} "
+        return f"{{{str(self)} | <{name}> | label:{self.label_idx} | path:{self.im_path}}}"
+
+
+class DetectionBbox(AnnotationBbox):
+    """ Inherits from AnnotationBbox """
+
+    def __init__(
+        self,
+        left: int,
+        top: int,
+        right: int,
+        bottom: int,
+        label_idx: int,
+        score: float,
+        im_path: str = None,
+        label_name: str = None,
+    ):
+        """ Initialize DetectionBbox """
+        super().__init__(
+            left,
+            top,
+            right,
+            bottom,
+            label_idx,
+            im_path=im_path,
+            label_name=label_name,
+        )
+        self.score = score
+
+    @classmethod
+    def from_array(
+        cls, arr: List[int], score: float, **kwargs
+    ) -> "DetectionBbox":
+        """ Create a Bbox object from an array [left, top, right, bottom]
+        This funciton must take in a score.
+        """
+        bbox = super().from_array(arr, **kwargs)
+        bbox.__class__ = DetectionBbox
```

Comment:
Should this line "bbox.__class__ = DetectionBbox" be in the constructor? Same comment applies also to the AnnotationBbox code